suites/test_search_beta.py

```diff
@@ -99,3 +99,38 @@ def modules_assemblies_list(self):
         lcc.log_info("Find the Search beta page preview in the attachment below:")
         self.driver.save_screenshot("search_beta_repo_selected.png")
         lcc.save_attachment_file("search_beta_repo_selected.png")
+
+    @lcc.test("Verify Status and Content Type filter on search page")
+    def filters_on_search_page(self):
+        utilities.click_element(self.driver, By.CLASS_NAME, locators.STATUS_FILTER_CLASS_NAME)
+        utilities.click_element(self.driver, By.CLASS_NAME, locators.DRAFT_STATUS_CLASS_NAME)
+        check_that("Draft Status filter chip is displayed on search page", utilities.find_element(self.driver, By.CLASS_NAME,
+                          locators.STATUS_TOOLBAR_CHIP_CLASS_NAME).is_displayed(), is_true())
+        last_published_date_list = utilities.find_elements_by_XPATH(self.driver, locators.LAST_PUBLISHED_DATE_XPATH)
+        published_date_list = []
+        for i in last_published_date_list:
+            published_date_list.append(i.text)
+        for i in published_date_list:
+            check_that("Draft status for all titles", i, contains_string("-"))
```

Comment:
Should this be something like "Published date for all titles contains '-' "?

Reply:
review changes acknowledged :) thanks for review @noopurmath3